src/commands/Moderation/Utilities/case-deprecations.ts

```diff
@@ -0,0 +1,47 @@
+import { LanguageKeys } from '#lib/i18n/languageKeys';
+import { SkyraCommand } from '#lib/structures';
+import { PermissionLevels, type GuildMessage } from '#lib/types';
+import { ApplyOptions } from '@sapphire/decorators';
+import { CommandOptionsRunTypeEnum } from '@sapphire/framework';
+import { send } from '@sapphire/plugin-editable-commands';
+import { chatInputApplicationCommandMention } from 'discord.js';
+
+@ApplyOptions<SkyraCommand.Options>({
+	name: '\u200Bcase-deprecations',
```

Comment:
why the zero-width space? Also i'm pretty sure we export a zws constant somewhere

Reply:
We haven't used them for the v7 deprecation commands, so I haven't used them either for this.